packages/app/src/background/shared/subworkers.js

```diff
@@ -0,0 +1,136 @@
+//
+// A workaround for resolving `Worker is not defined` error with MV3 on Chromium based platforms.
+(async function () {
+  if (global.chrome) {
```

Comment:
It's better to check if there is no chrome in global and just return at the start of the function.

```
if (!global.chrome) {
  return;
}
```